modules/widgets/top-posts.php

```diff
@@ -319,6 +319,22 @@ function widget( $args, $instance ) {
 			return;
 		}
 
+		/**
+		 * Filter the layout of the Top Posts Widget
+		 *
+		 * @module widgets
+		 * 
```

Comment:
Please also add a `since` tag as well, as done here:

https://github.com/Automattic/jetpack/blob/636bddb69109b405170426fbe957b924922f8cd9/modules/wordads/php/params.php#L8

Please use 6.4.0 as version.